msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/states/ResetPasswordStates.kt

```diff
@@ -233,25 +233,17 @@ class ResetPasswordCodeRequiredState internal constructor(
                     )
```

Comment:
I am not sure about the reason but should  INativeAuthCommandResult.Redirect be part of the possible results of ResetPasswordPasswordRequiredState.submitPassword

Reply:
+1 to Yuki. `ResetPasswordResendCodeCommandResult` shouldn't implement `Redirect` anymore, if the API can't return this state.
This will require changes in common.

Reply:
As discussed since this is not a breaking change and this also overlaps with the general internal enum class cleanup is not needed to be done now. I will create a PBI for our backlog